catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -346,7 +332,22 @@ protected Object executeWithSubject() throws Exception {
   }
 
   private File getInputFile() {
-    final File inputFile = new File(filePath);
+
+    final File inputFile;
+
+    if (StringUtils.isBlank(filePath)) {
+      inputFile = getDefaultIngestLocation();
+      if (!inputFile.exists()) {
+        printErrorMessage("No file or directory provided and no default directory exists.");
+        console.println("Provide file or directory as an argument to the command.");
+        return null;
```

Comment:
❓ I know this wasn't mentioned as part of the ticket, but would we ever want the default behaviour to try to ingest all files from the default directory when the file/path is missing?

Reply:
I guess I misread what was happening here. Ignore.

Reply:
Ignoring